From: Matteo Bernardini <ponce@slackbuilds.org>
Subject: Change header inclusion order to avoid xorg headers catching stdbool.h

libdrm commit e641e2a632d779f638ac2ba983b9fceb20b3fac4 added
stdbool.h to the library headers which conflicts with xorg headers.

diff -Naur xf86-video-qxl-0.1.5.orig/src/qxl_drmmode.c xf86-video-qxl-0.1.5/src/qxl_drmmode.c
--- xf86-video-qxl-0.1.5.orig/src/qxl_drmmode.c	2015-03-30 16:58:56.000000000 +0200
+++ xf86-video-qxl-0.1.5/src/qxl_drmmode.c	2022-02-10 08:59:13.221017000 +0100
@@ -33,7 +33,6 @@
 #ifdef XF86DRM_MODE
 
 #include <sys/ioctl.h>
-#include "qxl_drmmode.h"
 #include "X11/Xatom.h"
 #include "xf86DDC.h"
 /* DPMS */
@@ -47,6 +46,7 @@
 
 #include "qxl.h"
 #include "qxl_surface.h"
+#include "qxl_drmmode.h"
 
 static void drmmode_show_cursor (xf86CrtcPtr crtc);
 
